src/uproot_browser/plot.py

```diff
@@ -45,12 +45,14 @@ def plot_branch(tree: uproot.TBranch) -> None:
         ak.flatten(array) if array.ndim > 1 else array, bins=50, histogram=hist.Hist
     )
     plt.bar(histogram.axes[0].centers, histogram.values().astype(float))
+    return histogram
```

Comment:
Oops, Hist should know this is a Hist. Likely need to improve the typing for hist.numpy.histogram.

```suggestion
    return hist.Hist(histogram)
```